unix/README.md

```diff
@@ -102,31 +107,11 @@ directory:
 cd unix
 ```
 
-To install Machine Stats into an isolated environment we are going to use a
-tool to create such environments, called [`virtualenv`]. If you haven't
-already, install the tool using the information from the
-[Requirements](#Requirements) section.
-
-Let's create the virtual environment:
-
-```
-virtualenv venv
-```
-
-This will create a Python virtual environment under path `venv`.
-
-Now we need to activate the environment:
-
-```
-source venv/bin/activate
-```
-
-Now, when we have a Python virtual environment created and activated, let's
-install packages which are needed for Machine Stats to run properly by
+Let's install packages which are needed for Machine Stats to run properly by
```

Comment:
The diff on this hunk right here is beautiful. _so much simpler for people._ :clap: